packages/treasury/src/types.js

```diff
@@ -131,10 +145,9 @@
  * @param {ZCFMint} runMint
  * @param {Brand} collateralBrand
  * @param {ERef<PriceAuthority>} priceAuthority
- * @param {Rates} rates
- * @param {StageReward} rewardPoolStaging
+ * @param {GetParams} getLoanParams
+ * @param {ReallocateReward} reallocateReward
  * @param {TimerService} timerService
```

Comment:
I wonder if a promise works here:

```suggestion
 * @param {ERef<TimerService>} timerService
```

Reply:
yep.